libs/core/src/lib/menu/menu-keyboard.service.ts

```diff
@@ -2,6 +2,7 @@ import { Subject } from 'rxjs';
 import { Output, Injectable } from '@angular/core';
 import { DefaultMenuItem } from './default-menu-item';
 import { ListItemDirective } from '../list/list-item.directive';
+import { KeyUtil } from '../..';
```

Comment:
I don't know what is wrong with those auto applied imports. Let's change it to exact import, sometimes it throws errors